doc/getting_started/index.rst

```diff
@@ -161,95 +145,118 @@ You'll also install :ref:`west <west>`, Zephyr's multi-purpose tool.
 
       #. Open a new ``cmd.exe`` window **as a regular user** to continue.
 
-      #. Install west in the new window:
-
-         .. code-block:: bash
-
-            pip3 install west
-
 .. _Chocolatey: https://chocolatey.org/
 .. _Install chocolatey: https://chocolatey.org/install
 
 .. _get_the_code:
 .. _clone-zephyr:
+.. _install_py_requirements:
+.. _gs_python_deps:
 
 .. rst-class:: numbered-step
 
-Get the source code
-*******************
+Install Zephyr
+**************
 
-Clone Zephyr and its :ref:`modules <modules>` into a new :term:`west workspace`
-named :file:`zephyrproject`.
+Next, install Zephyr and its :ref:`modules <modules>` into a new :ref:`west
+<west>` workspace named :file:`zephyrproject`. You'll also install Zephyr's
+additional Python dependencies.
 
 .. tabs::
 
    .. group-tab:: Ubuntu
 
-      .. code-block:: bash
+      #. Install west, and make sure :file:`~/.local/bin` is on your
+         :envvar:`PATH` :ref:`environment variable <env_vars>`:
 
-         west init ~/zephyrproject
-         cd ~/zephyrproject
-         west update
+         .. code-block:: bash
 
-   .. group-tab:: macOS
+            pip3 install --user -U west
+            echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
+            source ~/.bashrc
 
-      .. code-block:: bash
+      #. Get the Zephyr source code:
 
-         west init ~/zephyrproject
-         cd ~/zephyrproject
-         west update
+         .. code-block:: bash
 
-   .. group-tab:: Windows
+            west init ~/zephyrproject
+            cd ~/zephyrproject
+            west update
 
-      .. code-block:: bat
+      #. Export a :ref:`Zephyr CMake package <cmake_pkg>`. This allows CMake to
+         load boilerplate code required for building Zephyr applications.
```

Comment:
Changing the changes in later commit in same PR.
So unfriendly of you :wink: 

```suggestion
      #. Export a :ref:`Zephyr CMake package <cmake_pkg>`. This allows CMake to
         automatically load boilerplate code required for building Zephyr applications.
```

Reply:
Sorry, I will try to be more friendly in the future. Comments now addressed.